### DISCUSSION TITLE
关于官方文档中开发版本安装链接失效问题

### DISCUSSION THREAD

**Discussion by KeanuReeves007** [vuejs-translations/docs-zh-cn#630] [General]
链接地址：https://vuejs.bootcss.com/guide/installation.html
![image](https://user-images.githubusercontent.com/118440462/211728588-bfd9628f-b2e0-4970-bd70-a0a98e8a1541.png)
![image](https://user-images.githubusercontent.com/118440462/211728619-baf21371-8c63-4409-950d-906eae3c36c0.png)
虽然是个无关紧要的事情，但是希望也能修复下

**Comment by wxsms** [comment]
vue2 官方文档地址为 v2.cn.vuejs.org，经查并无此问题。
